Assets/HoloToolkit/Input/Scripts/OnGazeEnterEvent.cs

```diff
@@ -1,21 +0,0 @@
-﻿using UnityEngine;
```

Comment:
True, this is an opinion of mine, but if i could just do something in the Editor, I'd much prefer it to coding.  But if doing something in the Editor effects the performance of my application vs coding it, I'll take the coding over editor ease of use.